akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -265,6 +282,17 @@ private[akka] class Shard(
     }
   }
 
+  def passivateIdleEntities(): Unit = {
+    log.debug("Passivating idle entities")
+    val deadline = System.nanoTime() - settings.passivateIdleEntityAfter.toNanos
+    lastMessageTimestamp.foreach {
+      case (entityId, lastMessageTimestamp) ⇒
+        // may be in lastMessageTimestamp but not be in refById because: ??? FIXME figure out
```

Comment:
?

Reply:
Had a test fail with NPE but never quite figured out in what situation it wouldn't be in `refById` here.